parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -198,6 +197,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(super) type Blocked<T: Config> = StorageValue<_, bool, ValueQuery>;
 
+	#[pallet::storage]
+	pub(super) type MilagroSyncCommittees<T: Config> =
```

Comment:
We shouldn't use the implementation name in the type name.

```suggestion
	pub(super) type SyncCommittees<T: Config> =
```